next.config.js

```diff
@@ -15,7 +15,10 @@ const moduleExports = () => {
     API_VERSION: isProduction
       ? process.env.API_VERSION
       : process.env.STAGE_API_VERSION,
-    HOST: isProduction ? process.env.HOST_NAME : process.env.STAGE_HOST_NAME,
+    HUBSPOT_ACCESS_TOKEN: isProduction
+      ? process.env.HUBSPOT_ACCESS_TOKEN
+      : process.env.STAGE_HUBSPOT_ACCESS_TOKEN,
+    HUBSPOT_ILS_LIST_ID: process.env.HUBSPOT_ILS_LIST_ID,
```

Comment:
I'm having trouble finding usage of this envar. Was this accidentally merged in?

Reply:
Thank you for pointing this out! The list ID is no longer required and has been removed (at 6bd1eae25c8a5496bee732a7c654e50084e3130c).

I also found some errors that needed to be addressed, so I adjusted the codebase and updated the "Purpose/Implementation Notes" section. This should help you get a better grasp of the changes, thank you David!